src/app/app.component.html

```diff
@@ -31,4 +31,12 @@
       <ion-router-outlet id="main-content"></ion-router-outlet>
     </div>
   </div>
+  <ion-footer *ngIf="footerDefaults.templateName">
```

Comment:
question(blocking)
If the footer is appearing below the content div will it be covering the bottom of the content on pages with full scroll - none of the demos show scrolling to the bottom to see if the last text is available. Please confirm